lib/ansible/module_utils/vmware.py

```diff
@@ -170,8 +170,8 @@ def find_datastore_by_name(content, datastore_name):
     return find_object_by_name(content, datastore_name, [vim.Datastore])
 
 
-def find_dvs_by_name(content, switch_name):
-    return find_object_by_name(content, switch_name, [vim.DistributedVirtualSwitch])
+def find_dvs_by_name(content, switch_name, folder=None):
```

Comment:
I don't understand why you need to change this since you don't call this sub later

Reply:
I've copied `lib/ansible/module_utils/vmware.py` directly from the Ansible repo, I think the OVH's one is not up to date...